Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor
+{
+    internal class BlenderLauncher : IDCCLauncher
+    {
+        const string menuItem_OpenInBlender = "Assets/MeshSync/Open in blender";
+
+        const string editorSettingPath = "MESHSYNC_BLENDER_PATH";
+
+        System.Diagnostics.Process m_blenderProcess;
+        static bool redirectBlenderToUnityConsole;
+
+        public RunMode runMode { get; set; }
+
+        ~BlenderLauncher()
+        {
+            Cleanup();
+        }
+
+        static string GetBlenderPath()
+        {
+            var blenderPath = EditorPrefs.GetString(editorSettingPath);
+
+            if (string.IsNullOrEmpty(blenderPath))
+            {
+                if (!DefaultAppUtility.TryGetRegisteredApplication(".blend", out blenderPath))
+                {
+                    if (Application.platform == RuntimePlatform.OSXEditor)
+                    {
+                        blenderPath = "/Applications/Blender.app/Contents/MacOS/Blender";
```

Comment:
Why is the BlenderPath hardcoded ?
My Blender path on Mac is like this
<img width="337" alt="image" src="https://user-images.githubusercontent.com/7379311/175883028-6088dff1-428b-4ec7-a1b0-ba4dbf73e2dd.png">


and I have the following error:
```
Win32Exception: Cannot find the specified file
System.Diagnostics.Process.StartWithShellExecuteEx (System.Diagnostics.ProcessStartInfo startInfo) (at <ef151b6abb5d474cb2c1cb8906a8b5a4>:0)
System.Diagnostics.Process.Start () (at <ef151b6abb5d474cb2c1cb8906a8b5a4>:0)
(wrapper remoting-invoke-with-check) System.Diagnostics.Process.Start()
Unity.MeshSync.Editor.BlenderLauncher.OpenDCCTool (UnityEngine.Object asset) (at Packages/com.unity.meshsync/Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs:159)
Unity.MeshSync.Editor.MeshSyncServerInspectorUtils.OpenDCCAsset (Unity.MeshSync.MeshSyncServer server) (at Packages/com.unity.meshsync/Editor/Scripts/MeshSyncServerInspectorUtils.cs:23)
Unity.MeshSync.Editor.BlenderLauncher.OpenBlender () (at Packages/com.unity.meshsync/Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs:80)
```

Reply:
So we are using `DefaultAppUtility.TryGetRegisteredApplication()` now. Nice.